<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>REGISTRATION</class>
 <widget class="QMainWindow" name="REGISTRATION">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>914</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="pushCapture">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>500</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Capture</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveButton">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>500</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="firstNameInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>150</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="lastNameInput">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>260</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="firstNameLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>130</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>FIRST NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="lastNameLabel">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>151</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>LAST NAME</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayVideo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="trainButton">
    <property name="geometry">
     <rect>
      <x>720</x>
      <y>500</y>
      <width>171</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>TRAIN</string>
    </property>
   </widget>
   <widget class="QLabel" name="warningText">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>390</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Sunken</enum>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
